CapitalForecasting/Modules/hierarchical_modeling.R

```diff
@@ -0,0 +1,44 @@
+library(tidyverse)
```

Comment:
I would like comments throughout the code to break up what you are doing throughout the code; that way it is easier to read it from an outsider's perspective.  For example, before the libraries you could write "# Libraries" then "# Loading Data", etc.